### PR TITLE
WIP: Add `social-media/publish-post` profile

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds `social-media/publish-post` profile with following providers

- [x] Instagram
- [x] Facebook

- [ ] Twitter - requires OAuth 1.0 authentication on create tweet endpoint, OAuth 1.0 auth header contains HMAC signature from request and auth parameters, we will have to add support for OAuth 1.0 to OneSDK to support Twitter

OAuth 1.0 Postman pre-request script example: https://github.com/twitterdev/postman-twitter-ads-api/blob/master/scripts/pre-request.js

❗  This PR is blocked by ONESDK support for setting security schema values and parameters as perform call arguments.

## Motivation and Context

User request: https://superfaceai.slack.com/archives/C02EXV8R546/p1638011134042000

I used parameters to pass `Instagram business account ID`/ `Facebook page ID`/ `Facebook page token` values. Those parameters are specific to Instagram / Facebook, but I am not sure if it is good solution. Any thoughts?

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: providers/instagram-graph.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "instagram-graph",
```
Comment (jnv) [inline providers/instagram-graph.json:2]: Not sure if it is necessary to distinguish "Instagram Graph API" from the old [legacy API](https://www.instagram.com/developer/) given the latter one has been definitely deprecated.

Comment (janhalama) [reply providers/instagram-graph.json]: I changed provider name to `instagram`.

File: grid/social-media/share-post/maps/instagram.suma [hunk 2]
```diff
@@ -0,0 +1,72 @@
+// Instagram Graph API Reference: https://developers.facebook.com/docs/instagram-api
+
+profile = "social-media/share-post@1.0"
+provider = "instagram"
+
+map SharePost {
+  apiVersion = 'v12.0'
```
Comment (Jakub-Vacek) [inline grid/social-media/share-post/maps/instagram.suma:7]: Shouldn't this be integration parameter?

Comment (janhalama) [reply grid/social-media/share-post/maps/instagram.suma:7]: I do not think version of API should be integration parameter. When new version is released we should release new version of map, generate new test snapshots and check that tests are passing.

**Comment by janhalama** [comment]
This PR is obsolete.

**Pull request closed without merging**